### PR TITLE
Update ward level specifications 

### PR DESCRIPTION
**Pull request by musamusa**

### CODE REVIEW

**Comment by musamusa** [comment]
@nicpottier kindly review

**Comment by nicpottier** [comment]
@rowanseymour plz review

File: java/src/test/java/io/rapidpro/flows/runner/RunnerTest.java [hunk 1]
```diff
@@ -232,4 +232,24 @@ public void updateContactField() throws Exception {
 
         assertThat(run.getContact().getFields().get("district"), is("Gasabo"));
     }
+
+    @Test
+    public void updateContacWardtField() throws Exception {
```
Comment (rowanseymour) [inline java/src/test/java/io/rapidpro/flows/runner/RunnerTest.java]: Typo

File: java/src/main/java/io/rapidpro/flows/definition/tests/location/HasWardTest.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+     * @see Test#fromJson(JsonElement, Flow.DeserializationContext)
+     */
+    public static HasWardTest fromJson(JsonElement elm, Flow.DeserializationContext context) throws FlowParseException {
+        JsonObject obj = elm.getAsJsonObject();
+        return new HasWardTest(obj.get("state").getAsString(), obj.get("district").getAsString());
+    }
+
+    @Override
+    public JsonElement toJson() {
+    return JsonUtils.object("type", TYPE, "state", m_state, "district", m_district);
```
Comment (rowanseymour) [inline java/src/main/java/io/rapidpro/flows/definition/tests/location/HasWardTest.java]: Formatting

**Comment by rowanseymour** [comment]
Besides a typo, looks good!

**Comment by musamusa** [comment]
@rowanseymour typo and formatting fixed

**Pull request merged**